src/instructlab/utils.py

```diff
@@ -239,6 +244,57 @@ def git_clone_checkout(
     return repo
 
 
+def get_confluence_docs(
+    logger,
+    source: Dict[str, Union[str, List[str]]],
+) -> List[str]:
+    """
+    Retrieve the content of Confluence documents
+
+    Args:
+        source (dict): Source info containing references
+
+    Returns:
+         List[str]: List of document contents.
+    """
+    cf = source["confluence"]
+    cfg = config.read_config()
```

Comment:
I don't think this is the correct approach. You have to pass the config from the `ctx` in `instructlab/lab.py` into the function, then access it with `ctx.obj.config`.